qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -173,3 +176,24 @@ def _create_qubo_matrix(self):
             x_index = [i for i, x in enumerate(lst_x) if x is True]
             if len(x_index) > 0:
                 raise QiskitOptimizationError('Pauli Xs exist in the Pauli {}'.format(pauli.x))
+
+    @property
+    def linear(self) -> bool:
+        r"""
+        Returns the conversion mode for quadratic terms
+
+        Returns:
+            If it's True :math:`x^2` is treated as a linear term.
+            Else, :math:`x^2` is treated as a quadratic term.
+        """
+        return self._linear
+
+    @linear.setter  # type:ignore
```

Comment:
Is this pylint `type: ignore` on the setter needed? I see it elsewhere on another setter so am wondering if it was inadvertently copied from somewhere where it was needed.